i18n/qqq.json

```diff
@@ -87,5 +87,7 @@
 	"powered-by-toolforge": "Label for the link to the Toolforge homepage. Displayed in the footer on every page.",
 	"privacy-policy": "Label for the link to the Wikimedia privacy policy. Displayed in the footer on every page.",
 	"report-issue": "Label for issue-reporting link to Phabricator. Displayed in the footer on every page.",
-	"view-source": "Label for the link to the Github project page. Displayed in the footer on every page."
+	"view-source": "Label for the link to the Github project page. Displayed in the footer on every page.",
+	"translatewiki": "Translate this tool. Displayed in the footer on every page."
```

Comment:
These are the instructions for translators, who can already see the "Translate this tool" base text. It's good to explain how the text is to be used (i.e. that it's a link).